packages/jaeger-ui/src/components/DeepDependencies/Graph/DdgNodeContent/index.tsx

```diff
@@ -111,6 +111,7 @@ export function getNodeRenderer({
   return function renderNode(vertex: TDdgVertex, _: unknown, lv: TLayoutVertex<any> | null) {
     const { isFocalNode, key, operation, service } = vertex;
     return (
+      // eslint-disable-next-line @typescript-eslint/no-use-before-define
```

Comment:
it's good to have a comment explaining the reason (I assume it's a false positive)